run_deterministic_tests.py

```diff
@@ -0,0 +1,337 @@
+"""
+This script is used to run deterministic tests on the Fastlane Bot.
+
+The script is run from the command line with the following command: `python run_deterministic_tests.py --task <task>
+--rpc_url <rpc_url> --network <network> --arb_mode <arb_mode>` --timeout_minutes <timeout_minutes> --from_block
+<from_block> --create_new_testnet <create_new_testnet>
+
+The `--task` argument specifies the task to run. The options are:
+- `set_test_state`: Set the test state based on the static_pool_data_testing.csv file.
+- `get_carbon_strategies_and_delete`: Get the carbon strategies and delete them.
+- `run_tests_on_mode`: Run tests on the specified arbitrage mode.
+- `end_to_end`: Run all of the above tasks.
+
+The `--rpc_url` argument specifies the URL for the RPC endpoint.
+
+The `--network` argument specifies the network to test. The options are:
+- `ethereum`: Ethereum network.
```

Comment:
no other options?

Reply:
more options are in the pipeline but not ready/usable yet